cmd/run.go

```diff
@@ -124,6 +125,7 @@ the data document with the following syntax:
 		},
 	}
 
+	runCommand.Flags().StringVarP(&params.PluginDir, "plugin-dir", "p", "", "set path to plugin directory")
```

Comment:
Do we need two flags? From the user's perspective, it's probably simpler to have a single directory that they can dump shared object files into and then OPA does the right thing.

Reply:
Sure.